static/js/hotkey.js

```diff
@@ -471,7 +471,8 @@ exports.process_hotkey = function (e, hotkey) {
             if (
                 ["#recent_topics", "#", ""].includes(window.location.hash) &&
                 !popovers.any_active() &&
-                !overlays.is_active()
+                !overlays.is_active() &&
+                !$("#searchbox_form #search_query").is(":focus")
```

Comment:
Since this bug also occurs for `stream-list-filter` and `user-list-filter` searchboxes.
Maybe we could update this condition to check if `document.activeElement.tagName` is not "input".

Reply:
Weird! I was not able to reproduce this for stream and user search. Are you able to reproduce it consistently? I am actually confused here a bit since when stream / user search is active `popovers.any_active()` is true here, I will try debugging more on this.

Reply:
Apparently `$("[class^='column-'].expanded").length` is 1 when user / stream search is active, hence popovers.any_active is true. So, ideally this bug should not be present in user / stream search. Can you please try reproducing on this branch and report?

Reply:
Hmm, I can't reproduce this consistently.
Occurs after a few tries, but once it occurs I can reproduce always.
This was on CZO. Will test on this branch and try to find a reproducer.

Reply:
Since this could be a flaky reproduce, I will add a fix for stream / user search explicitly anyway. Thanks!